scripts/benchmark/checks/deploy_dos.py

```diff
@@ -43,8 +43,8 @@
 )
 
 FIRST_TX_GAS_LIMIT = 367724
-SECOND_TX_GAS_LIMIT = 62050
-THIRD_TX_GAS_LIMIT = 104789
+SECOND_TX_GAS_LIMIT = 63042
```

Comment:
Had to increase gas price(`+992`) here in order to get the tests passing. Don't know how updating upstream tests affects benchmark tests. Is this expected or should I dig into this? cc @cburgdorf

Reply:
Istanbul reprices some opcodes so I'm *relatively* sure that this change in the tx gas limit is legit. :+1: